samples/net/syslog_net/src/main.c

```diff
@@ -36,6 +37,16 @@ int main(void)
 		const struct log_backend *backend = log_backend_net_get();
 
 		if (!log_backend_is_active(backend)) {
+
+			/* Specifying an address by calling this function will
+			 * override the value given to LOG_BACKEND_NET_SERVER.
+			   It can also be called at any other time. The net context
+			   will be released and restarted with the newly specified
+			   address.
+			 */
+			if (strlen(CONFIG_LOG_BACKEND_NET_SERVER) == 0) {
+				log_backend_net_set_addr("192.0.2.2:514");
```

Comment:
Please no hardcoded addresses as it makes it impossible to change the address without editing the file. Please add an option in sample Kconfig file if we need a default value to be used.